login.css

```diff
@@ -52,6 +52,10 @@ nav a{
 	text-align: center;
 }
 
+nav a:hover{
```

Comment:
12/08/2021: Adding the nav hover so when user hover to the Link the the background of it will turn to back and it display a better contrast for user.